src/main/java/uk/gov/hmcts/reform/sscs/model/docassembly/WriteFinalDecisionTemplateContent.java

```diff
@@ -182,44 +198,70 @@ protected String getOtherPartyNamesAttendedString(String otherPartyNamesAttended
-                return singletonList(getTriageConsideredParagraph(bundlePage));
+            if (appellantAttended) {
+                if (equalsIgnoreCase("faceToFace", hearingType)) {
+                    return singletonList("This has been an oral (face to face) hearing. "
+                            + getAppellantAndOtherPartyAttended(true,false, hearingType, appellantName, appointeeName, presentingOfficerAttended, bundlePage, otherPartyNamesAttended));
+                } else if (equalsIgnoreCase("triage", hearingType)) {
+                    return singletonList(getTriageConsideredParagraph(bundlePage));
+                } else {
+                    return singletonList("This has been a remote hearing in the form of a " + hearingType + " hearing. "
+                            + getAppellantAndOtherPartyAttended(true, false, hearingType, appellantName, appointeeName, presentingOfficerAttended, bundlePage, otherPartyNamesAttended));
+                }
             } else {
-                return asList("This has been a remote hearing in the form of a " + hearingType + " hearing. " + appellantName + " did not attend the hearing today. "
-                        + getPresentingOfficerAttendance(presentingOfficerAttended),
-                    getConsideredParagraph(bundlePage, appellantName));
+                if (equalsIgnoreCase("faceToFace", hearingType)) {
+                    return asList(appellantName + " the appellant requested an oral hearing but did not attend today. "
+                                    + getPresentingOfficerAttendance(presentingOfficerAttended),
+                            getConsideredParagraph(bundlePage, appellantName));
+                } else if (equalsIgnoreCase("triage", hearingType)) {
+                    return singletonList(getTriageConsideredParagraph(bundlePage));
+                } else {
+                    return asList("This has been a remote hearing in the form of a " + hearingType + " hearing. " + appellantName + " the appellant did not attend the hearing today. "
+                                    + getPresentingOfficerAttendance(presentingOfficerAttended),
+                            getConsideredParagraph(bundlePage, appellantName));
+                }
             }
         }
     }
 
     public void addHearingType(WriteFinalDecisionTemplateBody writeFinalDecisionTemplateBody) {
-        for (String hearingTypeSentence : getHearingTypeSentences(writeFinalDecisionTemplateBody.getAppellantName(), writeFinalDecisionTemplateBody.getPageNumber(),
-            writeFinalDecisionTemplateBody.getHearingType(), writeFinalDecisionTemplateBody.isAttendedHearing(), writeFinalDecisionTemplateBody.isPresentingOfficerAttended(),
+        for (String hearingTypeSentence : getHearingTypeSentences(writeFinalDecisionTemplateBody.getAppellantName(), writeFinalDecisionTemplateBody.getAppointeeName(), writeFinalDecisionTemplateBody.getPageNumber(),
```

Comment:
Might make more sense to just pass in the writeFinalDecisionTemplateBody

Reply:
I have passed the writeFinalDecisionTemplateBody into the parameter.